includes/autoload_func.php

```diff
@@ -44,13 +44,15 @@
             //echo 'objectMethod ' . $entry['class'] . "\n";
             $objectName = $entry['object'];
             $methodName = $entry['method'];
-            if (is_object($_SESSION[$objectName])) {
-                $_SESSION[$objectName]->$methodName();
-            }
-            if (!is_object($_SESSION[$objectName])) {
-                ${$objectName}->$methodName();
+            if (isset($_SESSION[$objectName])) {
+              if (is_object($_SESSION[$objectName])) {

```

Comment:
Couldn't lines 47-53 be reduced to a simple if-then-else rather than an if (condition) and an if (!condition)?

Reply:
Maybe but I only wanted to fix the one bug so that people could start using Report All Errors (Admin) again.

Reply:
Much appreciated.